kong/cmd/utils/nginx_signals.lua

```diff
@@ -18,15 +18,15 @@ local function is_openresty(bin_path)
   local cmd = fmt("%s %s", bin_path, nginx_version_command)
   local ok, _, _, stderr = pl_utils.executeex(cmd)
   if ok and stderr then
-    log.debug("%s: '%s'", cmd, stderr:sub(1, -2))
```

Comment:
Actually I think this is pretty useful and it's `debug`, not `verbose`, so it's always useful to have more insight as to what Kong is doing in its CLI. The same should be printed in debug too for other checks (`serf -v`, `resty -V`, dnsmasq.)